browser.js

```diff
@@ -15,22 +15,36 @@ const format = exports.format = require('././format');
  */
 exports.levels = require('././levels');
 
+/*
+ * @api private
+ * method {function} exposeFormat
+ * Exposes a sub-format on the main format object
+ * as a lazy-loaded getter.
+ */
+function exposeFormat(name, path) {
+  path = path || name;
+  Object.defineProperty(format, name, {
+    value: require(`./${path}.js`)
+  });
+}
+
 //
 // Setup all transports as eager-loaded exports
 // so that they are static for the bundlers.
 //
-Object.defineProperty(format, 'align',       { value: require('./align') });
-Object.defineProperty(format, 'cli',         { value: require('./cli') });
-Object.defineProperty(format, 'combine',     { value: require('./combine') });
-Object.defineProperty(format, 'colorize',    { value: require('./colorize') });
-Object.defineProperty(format, 'json',        { value: require('./json') });
-Object.defineProperty(format, 'label',       { value: require('./label') });
-Object.defineProperty(format, 'logstash',    { value: require('./logstash') });
-Object.defineProperty(format, 'metadata',    { value: require('./metadata') });
-Object.defineProperty(format, 'padLevels',   { value: require('./pad-levels') });
-Object.defineProperty(format, 'prettyPrint', { value: require('./pretty-print') });
-Object.defineProperty(format, 'printf',      { value: require('./printf') });
-Object.defineProperty(format, 'simple',      { value: require('./simple') });
-Object.defineProperty(format, 'splat',       { value: require('./splat') });
-Object.defineProperty(format, 'timestamp',   { value: require('./timestamp') });
-Object.defineProperty(format, 'uncolorize',  { value: require('./uncolorize') });
+exposeFormat('align');
+exposeFormat('cli');
+exposeFormat('combine');
+exposeFormat('colorize');
+exposeFormat('json');
+exposeFormat('label');
+exposeFormat('logstash');
+exposeFormat('metadata');
+exposeFormat('ms');
+exposeFormat('padLevels', 'pad-levels');
+exposeFormat('prettyPrint', 'pretty-print');
+exposeFormat('printf');
+exposeFormat('simple');
+exposeFormat('splat');
+exposeFormat('timestamp');
+exposeFormat('uncolorize');
```

Comment:
For this `browser.js` file the _repeated_ usage of `Object.defineProperty` is intentional. Bundlers like `webpack` are not intelligent enough to follow the function. I could be mis-remembering, but I recall there being a very specific reason why we didn't use a function here.